ingestion/src/metadata/ingestion/source/database/databricks/metadata.py

```diff
@@ -166,7 +166,7 @@ def get_columns(self, connection, table_name, schema=None, **kw):
         if col_type in {"array", "struct", "map"}:
             col_name = f"`{col_name}`" if "." in col_name else col_name
             try:
-                rows = dict(
```

Comment:
@kuma0128 I don't think we need to replace these once. this is python native `dict` and not pydantic `dict`. The pydantic `dict` would be something like `PydanticModel.dict()`